pkg/networkmanager/firewall.jsx

```diff
@@ -65,6 +65,8 @@ function ServiceRow(props) {
             { udp.map(p => p.port).join(', ') }
         </div>,
         <button className="btn btn-danger pficon pficon-delete"
+                title={props.readonly ? _("You are not authorized to modify the firewall.") : null}
```

Comment:
Also the wrong tooltip. Here it seems mostly irrelevant as an unprivileged user cannot even see the current firewall rules. But there is the theoretical possibility that an admin changes the polkit rules to allow that to users, but not modification.

Reply:
I can see the firewall rules as an unprivileged user on fedora. Fixed as above.